docs/openshift.asciidoc

```diff
@@ -0,0 +1,140 @@
+[id="{p}-openshift"]
+== Openshift
+
+This section will help you run ECK on Openshift.
+
+NOTE: Only Elasticsearch and Kibana are supported on Openshift.
+
+=== Prerequisites 
+
+. The following instructions must be run as the `system:admin` user or as a `User` with some privileges to create  `Projects`, `CRDs` and `RBACs` resources at the cluster level.
+
+. Set virtual memory settings on the nodes
++
+Before deploying an Elasticsearch cluster with ECK you must ensure that the `vm.max_map_count` setting is correctly applied on all the nodes of your cluster. This is because the Pods created by ECK are likely to run with the `restricted` SCC, they will run with a limited set of privileges and will not be able to tune this setting. You can refer to the following https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[documentation] for more information.
```

Comment:
> Before deploying an Elasticsearch cluster with ECK, make sure you correctly applied the `vm.max_map_count` setting on all the nodes of your cluster. You will not be able to tune this setting afterwards, as the Pods created by ECK often run with a limited set of privileges (`restricted` SCC). Check the https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[Virtual memory documentation] for more information.

Reply:
It may be worth to spell the SCC acronym - I searched for it but couldn't find anything relevant.